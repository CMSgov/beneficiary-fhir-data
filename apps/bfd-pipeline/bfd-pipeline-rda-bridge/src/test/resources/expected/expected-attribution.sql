INSERT INTO mbi_sets (set_type, mbi)
VALUES
('Dev', '1S00E00AA01');


INSERT INTO mbi_sets (set_type, mbi)
VALUES
('Small', '1S00E00AA01'),
('Small', '1S00E00AA03');


INSERT INTO mbi_sets (set_type, mbi)
VALUES
('Large', '1S00E00AA01'),
('Large', '1S00E00AA03'),
('Large', '1S00E00AA04'),
('Large', '1S00E00AA00');
