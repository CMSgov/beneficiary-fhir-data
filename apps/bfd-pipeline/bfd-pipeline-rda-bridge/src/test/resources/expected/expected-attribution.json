[
  [
    "1S00E00AA01",
    ],
  [
    "1S00E00AA01",
    "1S00E00AA03",
    ],
  [
    "1S00E00AA01",
    "1S00E00AA03",
    "1S00E00AA04",
    "1S00E00AA00",
    ]
]
